tests/test_extension_registry.py

```
@@ -0,0 +1,1632 @@
+                last_seen DATETIME DEFAULT CURRENT_TIMESTAMP
+            )
+        """)
+        
+        conn.commit()
+        conn.close()
+    
+    def test_data_persistence_across_sessions(self):
+        """Test that data persists across manager instances."""
+        # Create some data
+        cat_id = self.manager.create_category("Test Category", "Test", 1, True)
+        ext_id = self.manager.create_extension(".test", cat_id, "Test Extension", None, True, False, True, False, False)
```

Ruff (0.12.2)

Local variable `ext_id` is assigned to but never used

Remove assignment to unused variable `ext_id`

(F841)